src/data/path.ts

```diff
@@ -176,6 +177,11 @@ export class ReferencesPath {
       } else {
         support.warns.notSupportedClassNamespacedEndpointsForReferences();
       }
+      if(support.version){
+        if (isValidWeaviateVersion(support.version)){
```

Comment:
If `isValidWeaviateVersion` changes as per my comment below, this will need to change too 😁